ods_ci/tests/Tests/100__deploy/130__operators/150__dsp_operator/150__dsp_operator.robot

```diff
@@ -0,0 +1,72 @@
+*** Settings ***
+Documentation       Test Cases with regards to the DSP Operator
+
+Library             Collections
+Library             SeleniumLibrary
+Library             OpenShiftLibrary
+Resource            ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Resource            ../../../../Resources/OCP.resource
+Resource            ../../../../Resources/RHOSi.resource
+Resource            ../../tasks/Resources/RHODS_OLM/uninstall/uninstall.robot
+
+Suite Setup         Suite Setup
+Suite Teardown      Suite Teardown
+
+
+*** Test Cases ***
+Detect Pre-existing Install Of Argo Workflows And Block RHOAI Install
+    [Documentation]    Detect Pre-existing Install Of Argo Workflows And Block RHOAI Install
+    [Tags]                  Operator                ODS-2651                Tier3
+    Delete Argo Workflow Crd
+    ${return_code}          ${output}               Run And Return Rc And Output
+    ...                     oc apply -f ./ods_ci/tests/Resources/Files/argo/crd.workflows.yaml
+    Should Be Equal As Integers
+    ...                     ${return_code}
+    ...                     0
+    ...                     msg=${output}
+    Can Uninstall RHODS Operator
+    Should Be Equal As Integers ${return_code}      0
+    Open OperatorHub
+    Install ODH Operator
+    Apply DataScienceCluster CustomResource         default-dsc
+    Wait Until Keyword Succeeds
+    ...                     5 min
+    ...                     30s
+    ...                     Resource Status Should Be
+    ...                     oc get DataScienceCluster default-dsc -o json | jq '.status.conditions[] | select(.type=="data-science-pipelines-operatorReady") | .status'
+    ...                     CapabilityDSPv2Argo
+    ...                     "False"
+    Wait Until Keyword Succeeds
+    ...                     5 min
+    ...                     30s
+    ...                     Resource Status Should Be
+    ...                     oc get DataScienceCluster default-dsc -o json | jq '.status.conditions[] | select(.type=="data-science-pipelines-operatorReady") | .status'
```

Comment:
maybe you could replace these lines with a keyword which makes clear what it is expecting: "Something Should Be"